### ISSUE TITLE
Update DB with Description and Languages from UI

### ISSUE THREAD

**Issue by EricNMitchell** [computationalmystic/augur-group7#4]
Ideally we are looking to be able to use form input from the UI to take information from the project's Community Mangers that will be useful to update newcomers to the project on the projects overall status and goals, as well as the languages that are currently used. This information would be submitted from the UI and then updated into the database. Currently augur does not have any functionality that is similar to this that would allow form input from the frontend to update the data in the database.

**Comment by dtsrkb** [comment]
We are waiting to finish working on getting the information already in the database displayed before we start working on this. No progress towards this has been made so far as we are working on the other issues.

**Comment by dtsrkb** [comment]
I added the wontfix label to this issue because we have decided to take a different direction for our project. We will no longer be using the form information like we did before, but this issue is still related to the new features we are trying to implement.
